pkg/operator/controller/extension/admission/admission.go

```diff
@@ -102,6 +106,9 @@ func (d *deployment) createOrUpdateAdmissionRuntimeClusterResources(ctx context.
 				"priorityClassName": v1beta1constants.PriorityClassNameGardenSystem400,
 			},
 		},
+		"webhookConfig": map[string]any{
+			"ownerNamespace": virtualNamespace(extension).GetName(),
+		},
```

Comment:
I'd rather vote for making this a detail value belonging to the virtual cluster:

```suggestion
		"gardener": map[string]any{
			"virtualCluster": map[string]any{
				"namespace": virtualNamespace(extension).GetName(),
			},
		},
```